formatting and wording fixes (#16)
squ1rrel-ctf/squ1rrel.dev: _posts/2022-11-10-bluehens-otp.md

exec = run . parse
```

> Using the wonderful Glasgow Haskell Compiler we run the Wagon interpreter with `ghci interpreter.hs` and run our Wagon program as such

Reviewer: ```suggestion
Using the wonderful Glasgow Haskell Compiler, we run the Wagon interpreter with `ghci interpreter.hs`, and run our Wagon program like this:
```
